shop/shop/src/main/java/ro/msg/learning/shop/entities/Customer.java

```diff
@@ -0,0 +1,29 @@
+package ro.msg.learning.shop.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Builder
+@Table(name = "customer")
+
```

Comment:
sometimes there is space after annotations, sometimes not. I would go with the second choice. pay attention to code formatting